### DISCUSSION TITLE
Bullying10k数据集import时发生错误

### DISCUSSION THREAD

**Discussion by chenjq2** [fangwei123456/SpikingFlow#479] [Q&A] [answered]
为什么在from spikingjelly.datasets.bullying10k import Bullying10kClassification时会提示：ModuleNotFoundError: No module named 'spikingjelly.datasets.bullying10k'，我已经在vscode上安装了最新版本的spikingjelly

**Comment by fangwei123456** [comment, marked as answer]
`bullying10k`目前只在开发版本中，pip上还没有更新。可以直接安装开发版

pip install -qU git+https://github.com/fangwei123456/spikingjelly

**Comment by chenjq2** [comment]
![image](https://github.com/fangwei123456/spikingjelly/assets/69673679/fc5522e9-dd1e-4e01-a2c4-441cc1533f79)
并且我想问一下这里的省略号是是省略了什么内容呢?有时我想把原文件下载到本地但这里的省略号会报错

**Reply by fangwei123456** [reply]
`from ..` 表示从当前代码的上级目录导入

**Reply by chenjq2** [reply]
指的是引入_init_.py中的类吗？

**Reply by fangwei123456** [reply]
指的是`spikingjelly`这一级目录下。例如`..datasets`就是`spikingjelly.datasets`
